doc/sphinxman/source/add_tests.rst

```diff
@@ -30,108 +30,312 @@
 
 .. _`faq:add_tests`:
 
+==========
+Test Suite
+==========
+
+A test suite plays a vital role in open-source software use and development.
+
+* For a |PSIfour| user, tests provide models of inputs that should work
+  "as-is" and a searchable collection of syntax and capabilities.
+  The test suite also allows high-quality development snapshots of the
+  codebase to be built automatically for users.
+
+* For a CMS user, a test suite alongside installed |PSIfour| can be used to show that this piece is working in a complex software environment.
```

Comment:
```suggestion
* For a user who has |PSIfour| as part of a complex software environment, a test suite alongside installed |PSIfour| can be used to show that the ||PSIfour| piece is working.
```

The abbreviation "CMS" isn't standard outside of CMS, so let's use it only when necessary. If I hadn't needed to review the QCEngine paper, I'd have no clue what CMS was.